apps/monitoring/perftest/base/prometheus-values.yaml

```diff
@@ -1,24 +1,24 @@
 apiVersion: v1
 data:
-  slack-webhook: ENC[AES256_GCM,data:CXdPAHFAPrAv1g3zXCsWY1v3HRFMDqMSs+A/2TkjeOtUg7BApqDby/7UmPMjZcU1UKqnwS7O3LAp9Xhzz13+sT8zcdyJxXo9k0jHL/c28RTREpzMNED+Ep/Dq2M6vouHqZMPpBr6hsw=,iv:2uRNSA8Let9LUpf1rXJK24FNDIxLBYJfqbLhFs9UkzA=,tag:vuIFcEslZjFU7pX0H7B88g==,type:str]
+    slack-webhook: ENC[AES256_GCM,data:pst8kpEBUb5+geuY6JSw1sGKQxgRttuo8sNFfayRcPvMzpyO8xcvLz5vtoehbWem/MP3VAAfSWqengJ++PlqSePKD4SBe4rrqqczuRPA1vE84rRa1jCvhVVhYf/3F2/g4ND1+9+sfQU=,iv:t0RcgBJUkiX17eBySXIcfNyjYO0GLb/bLw5kGS4UHS8=,tag:6Bi4pIX6MAtDMmhIfA/bCQ==,type:str]
```

Comment:
probably need to re-indent the file, see
https://github.com/hmcts/cnp-flux-config#sops